content/rs/administering/troubleshooting/cluster-recovery.md

```diff
@@ -1,215 +1,146 @@
-addresses. For additional details, refer to
-[DNS]({{< relref "/rs/installing-upgrading/configuring/cluster-name-dns-connection-management/_index.md" >}}).
-
-### Recovering the databases
-
-After recovering all of the nodes in the cluster you can now recover the
-databases. Databases that did not have persistence configured are
-recovered with the same configuration they had in the old cluster, but
-will not contain any data. Databases that had persistence configured are
-recovered with the same configuration they had in the old cluster, and
-their entire data intact. The data recovery is carried out from the
-persistence files located in the persistent storage drives, which were
-mounted to the nodes in the old cluster and are now mounted to the new
-nodes.
-
-By default, the database recovery process looks for the database
-persistence files in the persistence storage location configured for the
-node. If the persistence files of the databases from the old cluster are
-not stored in the new node's persistence storage location (which is
-recommended for the recovery process), you must first map the recovery
-path of each node to the location of the old persistence files. To do
-so, run the `node <id> recovery_path set` command in rladmin. The
-persistence files for each database are located in the persistent
-storage path of the nodes from the old cluster, under the /redis
-directory.
-
-You can view which databases are recoverable by running:
-
-```src
-rladmin recover list
-```
-
-The status for each database can be either ready for recovery or missing
-files. An indication of missing files in any of the databases may result
-from either of the following:
-
-- The recovery path of the nodes has not been set appropriately
-- There is some issue with the persistence files (for example: lack of
-    permissions to read the files, missing files, or corrupted files)
-
-First, make sure that the recovery path is set correctly on all of the
-nodes in the cluster. If that does not resolve the issues, [contact our
-support team](mailto:support@redislabs.com).
-
-To recover all of the databases, run the command `rladmin recover all`.
-
-To recover the databases one by one, run the command `rladmin recover db <id|name>`.
-
-After recovering the databases, you can run `rladmin status`
-to verify that the recovered databases are now active.
-
-**Note**: If you are running the recovery process on the same machines
-that were used for the original cluster and there are still Redis
-processes running on the machine the database recovery process will
-fail. Make sure to kill all Redis processes before attempting to run the
-recovery process.
+    database persistence files.
+
+    {{% note %}}
+Make sure that the user redislabs has permissions to access the storage location
+of the configuration and persistence files on each of the nodes.
+    {{% /note %}}
+
+    If you use local persistent storage, place all the recovery files on each of the cluster nodes.
```

Comment:
"place all the recovery files" - only the relevant recovery files are required. I.e. the CCS for the first node, and the BDB backup files for the rest.

Reply:
Are the "database persistence files" just the backup files that you create with export or the scheduled backup process? What about AOF or snapshots?